Source/WebCore/platform/graphics/skia/GraphicsContextSkia.h

```diff
@@ -113,6 +113,11 @@ class WEBCORE_EXPORT GraphicsContextSkia final : public GraphicsContext {
             SkPaint::Cap cap { SkPaint::kButt_Cap };
             SkPaint::Join join { SkPaint::kMiter_Join };
         } m_stroke;
+
+        struct {
+            std::optional<DashArray> array { std::nullopt };
+            float offset { 0.0f };
+        } m_dash;
```

Comment:
Sorry I don't think I was clear enough. You were using a `std::optional` as a sentinel value as to whether or not to create a `SkDashPathEffect` when it feels like array. size() > 0 would work perfectly fine.

Reply:
Done